runtime/shibuya/src/lib.rs

```diff
@@ -1270,6 +1271,88 @@ impl pallet_unified_accounts::Config for Runtime {
     type WeightInfo = pallet_unified_accounts::weights::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+    // Of course it's not true for Shibuya, but SBY is worthless, a test token.
+    pub const NativeCurrencyId: CurrencyId = CurrencyId::ASTR;
+    // Aggregate values for one day.
+    pub const AggregationDuration: BlockNumber = 7200;
+}
+
+impl pallet_price_aggregator::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type MaxValuesPerBlock = ConstU32<8>;
+    type ProcessBlockValues = pallet_price_aggregator::MedianBlockValue;
+    type NativeCurrencyId = NativeCurrencyId;
+    // 7 days
+    type CircularBufferLength = ConstU32<7>;
+    type AggregationDuration = AggregationDuration;
+    type WeightInfo = pallet_price_aggregator::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    // Cannot specify `Root` so need to do it like this, unfortunately.
+    pub RootOperatorAccountId: AccountId = AccountId::from([0xffu8; 32]);
+}
+
+impl orml_oracle::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type OnNewData = PriceAggregator;
+    type CombineData = DummyCombineData<Runtime>;
+    type Time = Timestamp;
+    type OracleKey = CurrencyId;
+    type OracleValue = CurrencyAmount;
+    type RootOperatorAccountId = RootOperatorAccountId;
+    type Members = OracleMembership;
+    type MaxHasDispatchedSize = ConstU32<8>;
+    type WeightInfo = oracle_benchmarks::weights::SubstrateWeight<Runtime>;
+    #[cfg(feature = "runtime-benchmarks")]
+    type MaxFeedValues = ConstU32<2>;
+    #[cfg(not(feature = "runtime-benchmarks"))]
+    type MaxFeedValues = ConstU32<1>;
+}
+
+pub type OracleMembershipInstance = pallet_membership::Instance1;
+impl pallet_membership::Config<OracleMembershipInstance> for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type AddOrigin = EnsureRoot<AccountId>;
+    type RemoveOrigin = EnsureRoot<AccountId>;
+    type SwapOrigin = EnsureRoot<AccountId>;
+    type ResetOrigin = EnsureRoot<AccountId>;
+    type PrimeOrigin = EnsureRoot<AccountId>;
+
+    type MembershipInitialized = ();
+    type MembershipChanged = ();
+    type MaxMembers = ConstU32<16>;
+    type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+// TODO: get rid of this after running the benchmarks
```

Comment:
PR for orml-benchmarks has been accepted, but membership pallet update is still pending.

I've decided to keep this in Shibuya for the time being, until it can be removed.
No plan to add it to Shiden/Astar since it's good enough to benchmark on Shibuya.

Rephrased the comments to reflect that.